nl.sarine.gpxViewer.json

```diff
@@ -0,0 +1,150 @@
+{
+    "app-id" : "nl.sarine.gpxViewer",
```

Comment:
Looks like the ID `nl.sarine.gpx-viewer` is [used](https://github.com/DaveDavenport/gpx-viewer/blob/96bdc6524d30c03b331125fa157ee0c5a8d7450e/src/gpx-viewer.c#L1842) in the code. If you change flatpak ID to match it, it shouldn't require dbus permissions.

Reply:
Ouch! Thanks! Will rework that